docs/local_setup_bare_metal.md

```diff
@@ -11,4 +11,553 @@ nav_order: 14
+
+Please note that sometimes it is needed to fix some "side" errors like
+rebuilding `libyajl` by hands (it is located in `tmp-bootstrap/libyajl`
+subdirectory.
+
+Last check if binary has been produced can be made:
+
+```
+$ ./kcat -V
+
+kcat - Apache Kafka producer and consumer tool
+https://github.com/edenhill/kcat
+Copyright (c) 2014-2021, Magnus Edenhill
+Version 1.7.1-2-g338ae3 (JSON, Avro, Transactions, IncrementalAssign, JSONVerbatim, librdkafka 1.8.2 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
+```
+
+
+
+### Check connection to Kafka
+
+Now check if Kafkacat/kcat works as expected. We can use local broker that's been started already:
+
+```
+$ ./kcat -L -b localhost:9092
+
+Metadata for all topics (from broker -1: localhost:9092/bootstrap):
+ 1 brokers:
+  broker 0 at hpe-dl380pgen8-02-vm-15.hpe2.lab.eng.bos.redhat.com:9092 (controller)
+ 0 topics:
+```
+
+
+## Insights Results Aggregator installation and setup
+
+Now Insights Results Aggregator can be installed and setup.
+
+
+
+### Dependend packages installation
+
+Install Go:
+
+```
+# dnf install go
+```
+
+Check if Go was installed correctly:
+
+```
+$ go version
+
+go version go1.16.15 linux/amd64
+```
+
+
+
+### Insights Results Aggregator setup
+
+Clone the repository:
+
+```
+$ git clone https://github.com/RedHatInsights/insights-results-aggregator.git
+
+Cloning into 'insights-results-aggregator'...
+remote: Enumerating objects: 10761, done.
+remote: Counting objects: 100% (786/786), done.
+remote: Compressing objects: 100% (451/451), done.
+remote: Total 10761 (delta 469), reused 527 (delta 326), pack-reused 9975
+Receiving objects: 100% (10761/10761), 11.32 MiB | 21.07 MiB/s, done.
+Resolving deltas: 100% (7198/7198), done.
+```
+
+Build the service:
+
+```
+$ cd insights-results-aggregator
+$ make
+```
+
+Check if the service has been built correctly:
+
+```
+$ ./insights-results-aggregator --help
+
+Clowder is not enabled, skipping init...
+Clowder is disabled
+
+Command '--help' not found
+
+Aggregator service for insights results
+
+Usage:
+
+    ./insights-results-aggregator [command]
+
+The commands are:
+
+    <EMPTY>             starts aggregator
+    start-service       starts aggregator
+    help                prints help
+    print-help          prints help
+    print-config        prints current configuration set by files & env variables
+    print-env           prints env variables
+    print-version-info  prints version info
+    migration           prints information about migrations (current, latest)
+    migration <version> migrates database to the specified version
+```
+
+
+
+### Aggregator database setup
+
+First, empty database needs to be prepared in PostgreSQL:
+
+```
+# psql -U postgres
+
+Password for user postgres:
+psql (13.4)
+Type "help" for help.
+
+
+
+postgres=# create database aggregator;
+CREATE DATABASE
+postgres=# \q
+```
+
+Check if migration have not happed (yet):
```

Comment:
happened

Reply:
also maybe this step is not needed at all, because `migrate latest` is an idempotent operation

Reply:
Fixed.